assets/theme-editor.js

```diff
@@ -3,6 +3,18 @@ function hideProductModal() {
   productModal && productModal.forEach(modal => modal.hide());
 }
 
+function forceScriptReload() {
+  const scriptsToReload = document.querySelectorAll('[id^=AnimationsScript], [id^=EnableZoomOnHover]');
+  console.log('scripts', scriptsToReload);
+  if (!scriptsToReload.length) return;
+
+  scriptsToReload.forEach(script => {
+    const newScriptTag = document.createElement('script');
+    newScriptTag.setAttribute('src', script.src);
+    script.parentNode.replaceChild(newScriptTag, script);
+  });
+}
```

Comment:
Instead of this, could you just call `initializeAnimationTrigger` again?

If you wanted to more precise/targetted, you could update `initializeAnimationTrigger` to accept an optional query selector parameter that is a string. Then you could set up a second (or more) IntersectionObserver just for that section. E.g.

```
const containerEl = containerSelector ? containerClassName.querySelector(containerSelector) : document;
const animationTriggerElements = Array.from(
  containerEl.getElementsByClassName('animation__trigger')
);
```